.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
+    - cd daemon-scheduler; make; make release; cd ../
```

Comment:
I copied "cd ../" from our PR template, do you know if each line on the script takes you back to root?